docs/source/projects/architecture/content-architecture.rst

```diff
@@ -175,4 +179,4 @@ Below is a high level visual overview of project relationships. It is current as
 
 .. image:: /_static/_images/repos_map.png
```

Comment:
this link is broken! Not sure if the image file would be better moved to /projects/architecture/figs/ like the other images in this file. If not, the link below would fix it I think
```suggestion
.. image:: ../../_static/_images/repos_map.png
```

Reply:
I'm not seeing it broken on the main repo (and the [image is rendering on the documentation site](https://docs.jupyter.org/en/latest/projects/architecture/content-architecture.html#projects-overview)) so I'll leave this comment for now. I can always add the commit if I figure out why it was breaking for you.

Thanks for being observant!

Reply:
Ah, okay. It doesn't render for me when I view the rich diff within the PR (but all the other images do). I didn't test elsewhere. I guess it's a GitHub flaw?

Reply:
I'm seeing the same on the rich diff; weird! I totally appreciate you calling it out so we know what's up in the future.